streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -527,12 +527,10 @@ private void runLoop() {
             if (requiresPoll) {
```

Comment:
Can we get rid of this now? See comment further down, but i believe we will always poll so we might as well remove this.